run/GCClassic/createRunDir.sh

```diff
@@ -976,6 +981,7 @@ else
 	RUNDIR_VARS+="RUNDIR_DUSTDEAD_EXT='on '\n"
 	RUNDIR_VARS+="RUNDIR_SEASALT_EXT='on '\n"
 	RUNDIR_VARS+="RUNDIR_SOILNOX_EXT='on '\n"
+	RUNDIR_VARS+="RUNDIR_MEGAN_EXT='on '\n"
```

Comment:
It looks like RUNDIR_MEGAN_EXT is never set if using GCAP. Is that oversight?

Reply:
@lizziel: Yes it was.  Now fixed in 207c9c6.